### PR TITLE
Deduplicate and move configuration vars in a single place between Makefile and experiment.mk

### PR DESCRIPTION
**Pull request by mzabaluev**
Move the variables that are important for the Terraform configuration back to the `Makefile`. Deduplicate variable assignments in `experiment.mk` that are also present in `Makefile`. It's better to have the single place where these need to be defined, and I don't think the experiment parameters file is it.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,6 @@
 include experiment.mk
 
+DO_INSTANCE_TAGNAME=main-testnet
 DO_VPC_SUBNET=172.19.144.0/20
```
Comment (mzabaluev) [inline Makefile:4]: The idea to move these two variables together is that it's necessary to customize them both to deconflict QA testnet experiments running concurrently in the same project. If `experiment.mk` looks like a more appropriate place for this, let's move them both there.

**Comment by lasarojc** [comment]
I am good with the deduplication and agree that DO_VPC_SUBNET and DO_INSTANCE_TAGNAME should be together (it was my omission), but I think experiment.mk is the place to have, since DO_VPC_SUBNET is a also an experiment wide configuration.

**Comment by mzabaluev** [comment]
> I am good with the deduplication and agree that DO_VPC_SUBNET and DO_INSTANCE_TAGNAME should be together (it was my omission), but I think experiment.mk is the place to have, since DO_VPC_SUBNET is a also an experiment wide configuration.

I've done as suggested, thanks.

I'm not sure if `VERSION{,2}_TAG` and `VERSION{,2}_WEIGHT` should also be considered part of the experiment.

**Comment by lasarojc** [comment]
Good point. Probably. But we can move them there once we start doing multi-version tests.

**Review by lasarojc — APPROVED** [review]

**Pull request merged**